apps/cli/src/node/docker-compose-dev.yml

```diff
@@ -76,7 +76,7 @@ services:
             - ./.sunodo/image:/var/opt/cartesi/machine-snapshots/0_0:ro
 
     validator:
-        image: sunodo/rollups-node:0.4.1
+        image: sunodo/rollups-node:devel
```

Comment:
Can we use some kind of variable here? 

To differ when calling from a `sunodo-dev` alias and from a `sunodo` cli installed.